packages/devtools_app/lib/src/debugger/debugger_model.dart

```diff
@@ -2,14 +2,61 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the LICENSE file.
 
+import 'dart:convert';
+import 'dart:typed_data';
+
+import 'package:flutter/foundation.dart';
 import 'package:flutter/material.dart';
 import 'package:meta/meta.dart';
+import 'package:pedantic/pedantic.dart';
 import 'package:vm_service/vm_service.dart';
 
+import '../config_specific/logger/logger.dart';
+import '../globals.dart';
+import '../inspector/diagnostics_node.dart';
+import '../inspector/inspector_service.dart';
 import '../trees.dart';
 import '../ui/search.dart';
 import '../utils.dart';
 
+/// Whether to include properties and children surfaced through
+/// Diagnosticable objects as part of the generic Debugger view of an object.
+bool includeDiagnosticPropertiesInDebugger = true;
+
+/// Set to false as it is hard to avoid confusing overlap between the children
```

Comment:
nit: add an initial line to this dart doc "Whether to..."

Reply:
done